src/containers/Tenant/Diagnostics/Diagnostics.tsx

```diff
@@ -55,11 +56,12 @@ function Diagnostics(props: DiagnosticsProps) {
 
     const location = useLocation();
 
-    const queryParams = qs.parse(location.search, {
-        ignoreQueryPrefix: true,
-    });
+    const queryParams = new URLSearchParams(location.search);
+    for (const key of pagesQueryParams) {
+        queryParams.delete(key);
+    }
```

Comment:
Why do you delete params here? Maybe there is another way to delete unneeded params? Further it may be needed to remove params for some other tab as well, If we will be removing params this way, it will be a mess